app/controllers/guitar_gallery_controller.rb

```diff
@@ -0,0 +1,25 @@
+class GuitarGalleryController < ApplicationController
```

Comment:
Should it be `GuitarsController`? I would suggest changing also the filename to `guitars_controller.rb`

Reply:
It was GuitarsController, but I thought I could change it to something more meaningful like GuitarGallery because it is in fact a Gallery of Guitars haha
I remember we were talking about the naming convention, would it be better if it was: GuitarGalleriesController? Though It would be a bit misleading because there is only one Guitar Gallery 
WDYT? @Ron-Lavi

Reply:
I think that later you could also use the same controller to display specific info on a guitar (show action) and also update and delete a guitar, you will probably also have a Guitar model (rails model :P) so IMHO it makes more sense if this would be `GuitarsController`.

Reply:
Good point, i'll change it
Thanks @Ron-Lavi !